### ISSUE TITLE
Logging multiple lines with a single call.  Not possible?

### ISSUE THREAD

**Issue by webJose** [medikoo/log4#18]
Hello, me again.

I have a simple task that eludes me, and I don't know if the reason is this package or a limitation in cosole, or what.

I want to output using `log.warn()` a multi-line string.  Especifically, the stack trace of an error object.  I want it output in different lines, but all I can get is a single line with `\n` embeeded in it.

Is there a wait to output multiple lined-values from this package?  If so, may I trouble you for the how?

Thank you.

**Comment by medikoo** [comment]
@webJose this library doesn't limit in any way multi-lline output, you may confirm with `log.warn(new Error().stack)` (and [`log-node`](https://github.com/medikoo/log-node) writer) that multiple lines will be outputted.

**Comment by webJose** [comment]
Hi again.  The problem comes when a fomatter is specified:

```js
log.warn('%s', new Error().stack);
```

**Comment by medikoo** [comment]
@webJose indeed. It's actually on purpose, to meaningfully isolate arbitrary string input (it follows what Node.js does with `utils.inspect`). Still if not desired you can escape that behavior, see last point in documentation of formatter that `log-node` relies on: https://github.com/medikoo/cli-sprintf-format#cli-dedicated-sprintf-formatter

**Comment by webJose** [comment]
Ah, yes `%#s` does the trick, but loses the coloring.  Any way to retain coloring?

**Comment by medikoo** [comment]
In this case, string is treated as raw text (as if it's part of the first string argument), so eventual extra coloring, if needed should be added manually
